R/cochranecommon.R

```diff
@@ -508,6 +508,12 @@ CochraneCommon   <- function(jaspResults, dataset, options, type) {
     else
       selectedTitles <- sapply(indexing, function(inx) inx[["titleMetaAnalysis"]])[textSearchPositive & !textSearchNegative]
 
+    if (length(selectedTitles) == 0){
+      jaspResults[["selectedOverviewTable"]]$addFootnote(gettext(
+        "There is no matching meta-analysis title for the specified text search", symbol = "Note: "))
```

Comment:
```suggestion
      jaspResults[["selectedOverviewTable"]]$addFootnote(gettext(
        "There is no matching meta-analysis title for the specified text search"), symbol = "Note: ")
```

I believe this is what you wanted...

Reply:
oh thanks, good catch!